staging/src/k8s.io/apiserver/pkg/endpoints/apiserver_test.go

```diff
@@ -1144,8 +1144,7 @@ func TestList(t *testing.T) {
 			t.Logf("%d: body: %s", i, string(body))
 			continue
 		}
-		// TODO: future, restore get links
-		if !selfLinker.called {
+		if utilfeature.DefaultFeatureGate.Enabled(features.RemoveSelfLink) == selfLinker.called {
 			t.Errorf("%d: never set self link", i)
```

Comment:
this error message is no longer correct for all the ways you could get into this if statement.

Reply:
done